src/main/java/org/folio/services/instance/InstanceService.java

```diff
@@ -124,7 +125,12 @@ public Future<Response> updateInstance(String id, Instance newInstance) {
     return refuseLongNotes(newInstance)
       .compose(notUsed -> instanceRepository.getById(id))
       .compose(CommonValidators::refuseIfNotFound)
-      .compose(oldInstance -> refuseWhenHridChanged(oldInstance, newInstance))
+      .compose(oldInstance -> {
+        if (!newInstance.getSource().startsWith("CONSORTIA")) {
```

Comment:
I need to clarify this because in [Sharing Instances With Consortium Members](https://wiki.folio.org/display/DD/Sharing+Instances+With+Consortium+Members) I see only "CONSORTIUM-*". Thanks

Reply:
I clarified. "-shared" is UI constant only.